planner/core/rule_join_elimination.go

```diff
@@ -76,7 +79,7 @@ func (o *outerJoinEliminator) isAggColsAllFromOuterTable(outerPlan LogicalPlan,
 	for _, col := range aggCols {
 		columnName := &ast.ColumnName{Schema: col.DBName, Table: col.TblName, Name: col.ColName}
 		if c, err := outerPlan.Schema().FindColumn(columnName); c == nil {
-			log.Warn(err)
+			logutil.Logger(context.Background()).Warn("outerJoinEliminator raises an error", zap.Error(err))
```

Comment:
there are three places which logs `outerJoinEliminator raises an error`. Could you also print the step information during the outer join elimination to help identify the problems?

Reply:
These warning logs should have been removed by https://github.com/pingcap/tidb/pull/9631.